projects/observability/src/shared/dashboard/widgets/charts/cartesian-widget/cartesian-widget-renderer.component.ts

```diff
@@ -46,22 +57,28 @@ export class CartesianWidgetRendererComponent<TData> extends InteractiveDataWidg
 
   public selectedInterval?: IntervalValue;
   public intervalOptions?: IntervalValue[];
-  private fetcher?: MetricSeriesFetcher<TData>;
+  private seriesFetcher?: MetricSeriesFetcher<TSeriesInterval>;
+  private bandFetcher?: MetricBandFetcher<MetricTimeseriesInterval>;
 
   public onIntervalChange(interval: IntervalValue): void {
     this.selectedInterval = interval;
     this.updateDataObservable();
   }
 
-  protected fetchData(): Observable<Series<TData>[]> {
-    return this.model.getSeriesFetcher().pipe(
-      tap(fetcher => {
-        this.fetcher = fetcher;
-        if (this.intervalSupported(fetcher)) {
+  protected fetchData(): Observable<CartesianData<TSeriesInterval, MetricTimeseriesBandInterval>> {
+    return forkJoinSafeEmpty({
+      seriesFetcher: this.model.getSeriesFetcher(),
+      bandFetcher: this.model.getBandsFetcher()
+    }).pipe(
+      tap(fetchers => {
+        this.seriesFetcher = fetchers.seriesFetcher;
+        this.bandFetcher = fetchers.bandFetcher;
+
+        if (this.intervalSupported(this.seriesFetcher)) {
           this.intervalOptions = this.buildIntervalOptions();
           this.selectedInterval = this.getBestIntervalMatch(
             this.intervalOptions,
-            this.selectedInterval || fetcher.getRequestedInterval()
+            this.selectedInterval || this.seriesFetcher.getRequestedInterval()
```

Comment:
if we're not asking the bands for intervals, they shouldn't be supported in their models.